lib/static/new-ui/utils/page.ts

```diff
@@ -0,0 +1,25 @@
+import {Page, PathNames} from '@/constants';
+
+export function getPageByPathname(pathname: string): Page {
+    if (pathname.startsWith(PathNames.visualChecks)) {
+        return Page.visualChecksPage;
+    }
+
+    if (pathname.startsWith(PathNames.suites)) {
+        return Page.suitesPage;
+    }
+
+    return Page.suitesPage;
+}
+
+export const getPathnameByPage = (page: Page): string => {
+    if (page === Page.visualChecksPage) {
+        return PathNames.visualChecks;
+    }
+
+    if (page === Page.suitesPage) {
+        return PathNames.suites;
+    }
+
+    return PathNames.suites;
```

Comment:
Redundant check: whats the point of "page === Page.suitesPage", if we return "PathNames.suites" in both cases?

I would write this function like this:
```js
return page === Page.visualChecksPage ? PathNames.visualChecks : PathNames.suites;
```